packages/expo/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -235,6 +321,37 @@ open class ExpoAppDelegate: ExpoAppInstance {
       return subscriber.application?(application, continue: userActivity, restorationHandler: handler) ?? false || result
     }
   }
+#elseif os(macOS)
+  open override func application(
+    _ application: NSApplication,
+    continue userActivity: NSUserActivity,
+    restorationHandler: @escaping ([any NSUserActivityRestoring]) -> Void
+  ) -> Bool {
+    let selector = #selector(application(_:continue:restorationHandler:))
+    let subs = ExpoAppDelegateSubscriberRepository.subscribers.filter { $0.responds(to: selector) }
+    var subscribersLeft = subs.count
+    let dispatchQueue = DispatchQueue(label: "expo.application.continueUserActivity", qos: .userInteractive)
+    var allRestorableObjects = [UIUserActivityRestoring]()
```

Comment:
`UIUserActivityRestoring` is that correct? looks like a uikit thing

Reply:
Good catch, we alias UIUserActivityRestoring to NSUserActivityRestoring in macOS but given that this is a macOS specific function we should use NSUserActivityRestoring